### PR TITLE
Update Hall of Fame

### PR DESCRIPTION
**Pull request by vixentael**
Thank you @maxammann for your contributions!

### CODE REVIEW

File: themis/community/credits.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ As for 2020, Themis is supported by core crew:
 
 A special thank you goes to the awesome contributors that help us make Themis better:
 
+- [@maxammann](https://github.com/maxammann) for improvements in wasmThemis code, documentation and triggering fire discussions ([#779](https://github.com/cossacklabs/themis/issues/779), [#781](https://github.com/cossacklabs/themis/pull/781), [#783](https://github.com/cossacklabs/themis/pull/783))
```
Comment (vixentael) [inline themis/community/credits.md:26]:
```suggestion
- [@maxammann](https://github.com/maxammann) for improvements in wasmThemis code, documentation and triggering fire discussions ([#779](https://github.com/cossacklabs/themis/issues/779), [#781](https://github.com/cossacklabs/themis/pull/781), [#783](https://github.com/cossacklabs/themis/pull/783)).
```

File: themis/community/credits.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ As for 2020, Themis is supported by core crew:
 
 A special thank you goes to the awesome contributors that help us make Themis better:
 
+- [@maxammann](https://github.com/maxammann) for improvements in wasmThemis code, documentation and triggering fire discussions ([#779](https://github.com/cossacklabs/themis/issues/779), [#781](https://github.com/cossacklabs/themis/pull/781), [#783](https://github.com/cossacklabs/themis/pull/783)).
```
Comment (ilammy) [inline themis/community/credits.md:26]: More common capitalization, Oxford comma, and you probably meant “flame” there but it's not really *flame* – as in people throwing flagrant insults left and right on the internet:
```suggestion
- [@maxammann](https://github.com/maxammann) for improvements in WasmThemis code, documentation, and triggering discussions ([#779](https://github.com/cossacklabs/themis/issues/779), [#781](https://github.com/cossacklabs/themis/pull/781), [#783](https://github.com/cossacklabs/themis/pull/783)).
```

**Review by ilammy — APPROVED** [review]

**Comment by maxammann** [comment]
Thank you very much! :) Very much appreciate how active you are here on github!

File: themis/community/credits.md (hunk 2 above)
Comment (maxammann) [reply themis/community/credits.md:26]: Hall of Fire :P

**Comment by vixentael** [comment]
@maxammann 

I made changes live, here https://docs.cossacklabs.com/themis/community/credits/ and #36 (https://docs.cossacklabs.com/themis/languages/wasm/installation/#building-latest-version-from-source).

🔥

**Pull request merged**